lib/vanagon/platform/windows.rb

```diff
@@ -1,6 +1,7 @@
 class Vanagon
   class Platform
     class Windows < Vanagon::Platform
+      attr_accessor :proj_root
```

Comment:
That sounds like a fair proposal.

![279694_5](https://cloud.githubusercontent.com/assets/344926/13368353/814beffe-dc9e-11e5-81ba-3c0acba09c86.jpg)